src/cli/cmds/fleet.toit

```diff
@@ -429,6 +429,74 @@ create-fleet-commands config/Config cache/Cache ui/Ui -> List:
       --run=:: create-reference it config cache ui
   cmd.add create-reference-cmd
 
+  migration-cmd := cli.Command "migration"
+      --aliases=["migrate"]
+      --help="""
+        Migrate a fleet to a new broker.
+
+        - Add a new broker to your configuration. Use 'config broker add ...'.
+        - Start the migration with 'migration start --broker=<broker>'.
+        - Upload a new pod, and roll it out to the fleet with 'fleet roll-out'.
+        - Check the status of the migration with the 'status' command.
+        - Finish the migration with 'migration finish', once all devices have migrated.
```

Comment:
Maybe this should be stop? Finish seems to imply that we're done with it better than stop, but on the other hand, we don't really know if we're done. If finish checked the status and complained if it wasn't really finished (unless --force), then maybe finish would be better.

All in all, I think start/stop is likely the simpler pair and we still check that we don't stop an incomplete migration without a '--force' flag.

Reply:
done.